### PR TITLE
Acceleration bands

### PR DESCRIPTION
**Pull request by Jay-Jay-D**
Implementation of the Acceleration bands. #1066

### CODE REVIEW

File: Indicators/AccelerationBands.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+        /// <summary>
+        ///     Gets the upper acceleration band  (High * ( 1 + Width * (High - Low) / (High + Low)))
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> UpperBand { get; }
+
+        /// <summary>
+        ///     Gets the lower acceleration band  (Low * (1 - Width * (High - Low)/ (High + Low)))
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> LowerBand { get; }
```
Comment (MartinMoli) [inline Indicators/AccelerationBands.cs]: all of these could be readonly?

Comment (Jay-Jay-D) [reply Indicators/AccelerationBands.cs]: Yes, but I followed the same criteria used in the BolligerBands indicator.

File: Tests/Indicators/AccelerationBandsTests.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            var abands = CreateIndicator();
+            TestHelper.TestIndicator(
+                abands,
+                "spy_acceleration_bands_20_4.txt",
+                "UpperBand",
+                (ind, expected) => Assert.AreEqual(expected, (double) ((AccelerationBands) ind).UpperBand.Current.Value,
+                    delta: 1e-4, message: "Upper band test fail.")
+            );
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Jay-Jay-D) [reply Tests/Indicators/AccelerationBandsTests.cs]: Please check the ´CommonIndicatorTests´ class.

File: Indicators/AccelerationBands.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            MiddleBand.Update(new IndicatorDataPoint
+            {
+                Time = input.Time,
+                Value = input.Close
+            });
+
+            LowerBand.Update(new IndicatorDataPoint
+            {
+                Time = input.Time,
+                Value = input.Low * (1 - _width * (input.High - input.Low) / (input.High + input.Low))
```
[the remark this answers is not in the document]
Comment (Jay-Jay-D) [reply Indicators/AccelerationBands.cs]: Done.

**Comment by jaredbroad** [comment]
Nice work @Jay-Jay-D, thanks for the review @MartinMoli !

**Pull request merged**